integration/src/main/java/scripts/utils/GcsBucketAccessTester.java

```diff
@@ -138,6 +146,8 @@ public void checkAccess(TestUserSpecification testUser, @Nullable ControlledReso
         assertTrue(doWithOptionalWait(this::blobUpdate), "editor can update");
         assertTrue(doWithOptionalWait(this::blobDelete), "editor can delete");
         assertTrue(doWithOptionalWait(this::bucketGet), "editor can bucket get");
+        // Do not wait for permissions which won't appear
+        needToWait = false;
```

Comment:
why do we need to call doWithOptionalWait if we are not waiting? can't we just do